app/controllers/registrations_controller.rb

```diff
@@ -1,5 +1,13 @@
 class RegistrationsController < Devise::RegistrationsController
 
+   # temporary function for adding company to new user
+  def create
+    super do
+      resource.company = Company.find(1)
```

Comment:
use load_and_authorize method instead of loading with specific id

Reply:
This will be resolved after merge